bucket/redream-ai.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": "1.10",
```

Comment:
Latest version in GitHub seem to be [1.12](https://github.com/Fictiverse/Redream/releases/tag/1.12), please update to latest version

Reply:
updated, thanks for suggestions!